.github/workflows/test-workflow.yml

```diff
@@ -62,3 +71,54 @@ jobs:
         cluster: github-actions-deploy-task-def-integ-test
         wait-for-service-stability: true
         enable-ecs-managed-tags: true
+
+  deploy_from_workflow_call:
```

Comment:
[The `uses` function does not support contexts](https://stackoverflow.com/questions/75373413/reference-a-variable-in-uses-when-pointing-to-a-path-to-a-container) (i.e. variables).

Thus, we are not able to dynamically call either a released action (e.g. `uses:aws-actions/amazon-ecs-deploy-task-definition@v1`) or a local action (e.g. `uses:./`) based on state. As a workaround, we opt to use separate jobs.